README.md

```diff
@@ -54,7 +54,7 @@ You can use the following resources to learn Ballerina.
 
 ## Report issues and security flaws
 
->**Tip:** If you are unsure whether you have found a bug, search the existing issues in the GitHub repo and raise it in the [Ballerina Discord](https://discord.com/invite/wAJYFbMrG2) or [Stack Overflow](https://stackoverflow.com/questions/tagged/ballerina).
+>**Tip:** If you are unsure whether you have found a bug, search the existing issues in the GitHub repo and raise them in the [Ballerina Discord](https://discord.com/invite/wAJYFbMrG2) or [Stack Overflow](https://stackoverflow.com/questions/tagged/ballerina).
```

Comment:
The `it` here seems to be referring to the bug the user has found. In that case the previous version was okay IMO

Reply:
Yeah, shall we leave this as is?